Document `offline mode` feature for workspace (#6032)
Scille/parsec-cloud: docs/userguide/workspace_offline_mode.rst

.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS

.. _doc_userguide_workspace_offline_mode:

Offline mode for Workspace
==========================

> In Parsec, you can enable offline mode for a workspace. This will allow you to access the workspace's files even if you are not connected to the internet.

Reviewer: ```suggestion
The offline mode allows you to access the workspace data even if you are not connected to the server.
```

> To enable or disable the offline mode, you need to open the workspace context menu by right-clicking on the workspace and click on the "Manage offline availability" button.

Reviewer: ```suggestion
To enable or disable the offline mode, right-click on a workspace and select ``Manage offline availability`` from the context menu.
```

.. image:: screens/workspace_context_menu.png
   :alt: Workspace context menu

> This will open a modal where you can enable or disable the offline mode (see images below).

Reviewer: ```suggestion
From the dialog, enable or disable offline mode by clicking the toggle button ``Offline availability``.
```

> - When offline mode is disabled (this is the default mode):

Reviewer: ```suggestion
- Offline mode is disabled:
```

  .. image:: screens/workspace_offline_modal_disabled.png
    :alt: Offline mode disabled

> - When offline mode is enabled:

Reviewer: ```suggestion
- Offline mode is enabled:
```

  .. image:: screens/workspace_offline_modal_enabled.png
    :alt: Offline mode enabled

> On both modals, you have the toggle button ``Offline availability`` to enable or disable the offline mode.

Reviewer: ```suggestion
In both cases, the dialog displays the amount of data available locally and the total amount of data in the workspace.
```
